doc/team.md

```diff
@@ -1,71 +1,61 @@
-# Team Gallery
+# PyGMT Team
 
-We are an international team dedicated to building a Pythonic API for the
-Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
-new and experienced users by creating user-friendly interfaces with the GMT
-C API, supporting rich display in Jupyter notebooks, and integrating with
-the PyData ecosystem.
+We are an international team dedicated to building a Pythonic API for the Generic Mapping
+Tools (GMT).
 
-All are welcome to become involved with the PyGMT project! For more information
-about how to get involved, see the {doc}`contributing`.
+All are welcome to become involved with the PyGMT project! For more information about how
+to get involved, see the {doc}`contributing`.
 
-## Distinguished Contributors
+PyGMT Distinguished Contributors are recognized for their substantial contributions to PyGMT,
+which may include code, documentation, pull request review, triaging, forum responses,
+community building and engagement, outreach, and inclusion and diversity. New Distinguished
+Contributors are selected twice per year by those listed below. Distinguished Contributors
+include current or inactive contributors and inactive maintainers, so is not meant as a means
+of conveying responsibilities. Distinguished Contributors who are also active maintainers of
+the PyGMT project have responsibilities detailed in the {doc}`maintenance`.
```

Comment:
I am trying to rephrase this paragraph. 

The first paragraph explains the different roles and the second paragraph documents the policy. OK with the changes? Feel free to rephrase in any way you like:
```suggestion
Distinguished Contributors are recognized for their substantial contributions to PyGMT,
which may include code, documentation, pull request review, triaging, forum responses,
community building and engagement, outreach, and inclusion and diversity. Maintainers
are recognized for their responsibilities in maintaining the project, as detailed in
{doc}`maintenance`.

New Distinguished Contributors and Active Maintainers are selected and voted by current
Active Maintainers before each release. Maintainers that are inactive for more than one
year will be moved to Distinguished Contributors.
```

Reply:
So far this sounds good to me!